docs/.vuepress/dist/404.html

```diff
@@ -22,12 +22,12 @@
+    <title>Pulsar Edit</title><meta name="description" content="Next generation hackable editor for the future">
+    <link rel="modulepreload" href="/assets/app.53920281.js"><link rel="modulepreload" href="/assets/404.html.ab8adbf7.js"><link rel="modulepreload" href="/assets/404.html.d2dc3323.js"><link rel="prefetch" href="/assets/about.html.c519d3c5.js"><link rel="prefetch" href="/assets/index.html.17987fb7.js"><link rel="prefetch" href="/assets/index.html.9dff0173.js"><link rel="prefetch" href="/assets/index.html.8301636d.js"><link rel="prefetch" href="/assets/index.html.8a2990a7.js"><link rel="prefetch" href="/assets/index.html.6deaee6d.js"><link rel="prefetch" href="/assets/index.html.2d4359e0.js"><link rel="prefetch" href="/assets/index.html.166ead40.js"><link rel="prefetch" href="/assets/list.html.f1df60fc.js"><link rel="prefetch" href="/assets/release-process.html.a3ab0cc6.js"><link rel="prefetch" href="/assets/autocomplete-providers.html.8e364812.js"><link rel="prefetch" href="/assets/index.html.3db0f550.js"><link rel="prefetch" href="/assets/provider-api.html.5a5dd9a4.js"><link rel="prefetch" href="/assets/symbolprovider-config-api.html.a1b17ad8.js"><link rel="prefetch" href="/assets/index.html.7944536c.js"><link rel="prefetch" href="/assets/june-2017.html.ea21c5e0.js"><link rel="prefetch" href="/assets/incomplete-classpath-warning.html.2111deaa.js"><link rel="prefetch" href="/assets/index.html.9b877383.js"><link rel="prefetch" href="/assets/about.html.a17bb8a5.js"><link rel="prefetch" href="/assets/index.html.5d4d364e.js"><link rel="prefetch" href="/assets/index.html.2ff11c7a.js"><link rel="prefetch" href="/assets/index.html.2e731043.js"><link rel="prefetch" href="/assets/index.html.3cd30915.js"><link rel="prefetch" href="/assets/index.html.331a7f41.js"><link rel="prefetch" href="/assets/index.html.38fe4b9f.js"><link rel="prefetch" href="/assets/index.html.44c5372b.js"><link rel="prefetch" href="/assets/list.html.c9176043.js"><link rel="prefetch" href="/assets/release-process.html.33321cd8.js"><link rel="prefetch" href="/assets/autocomplete-providers.html.a0c1e432.js"><link rel="prefetch" href="/assets/index.html.db62e0b4.js"><link rel="prefetch" href="/assets/provider-api.html.9c084256.js"><link rel="prefetch" href="/assets/symbolprovider-config-api.html.deabe7b0.js"><link rel="prefetch" href="/assets/index.html.7d77e4bf.js"><link rel="prefetch" href="/assets/june-2017.html.9625907a.js"><link rel="prefetch" href="/assets/incomplete-classpath-warning.html.b10d30db.js"><link rel="prefetch" href="/assets/index.html.31f8388e.js">
```

Comment:
What's the criteria for prefetching here? Does this come from VuePress or is this a manual change? If the latter is true, has the change been tested to see if it's beneficial for performance?

Reply:
No clue, that's not something I did. Unless there's a change I should make w/in vuepress' config somehow.